exercises/practice/anagram/AnagramTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
+declare (strict_types = 1);
```

Comment:
Please do not add spaces here (see CI output).

```suggestion
declare(strict_types=1);
```